apps/designer-standalone/src/app/AzureLogicAppsDesigner/DesignerCommandBar.tsx

```diff
@@ -49,14 +52,16 @@ export const DesignerCommandBar = ({
     updateCallbackUrl(designerState, DesignerStore.dispatch);
   });
 
-  const allOperationErrors = useSelector((state: RootState) => {
+  const designerIsDirty = useIsDesignerDirty();
```

Comment:
Do we need this as well for vscode?

Reply:
All of these command bar changes are just for standalone editor, but yes the save button should be disabled when designer isn't dirty

Reply:
Ok, I can update vscode side